index.md

```diff
@@ -1,23 +1,63 @@
-_“Best practice templates and writing instructions for documenting open source software.”_
+TheGoodDocsProject* exists because every project deserves good docs, and we believe that templates make good docs. 
 
-Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. How do you structure docs? What does a docs website look like? How do I keep docs consistently formatted? The list of questions far outweigh the answers.
+At Write The Docs meetups and conferences, developers frequently ask where to get started, or what kind of documentation to produce for a new project. 
 
-That’s where _The Good Docs Project_ fits in. Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of _Good Docs_ using a suite of templates. 
+We want to unblock developers (and even technical writers) by providing the necessities for documenting open source software. 
 
-Your _Good Docs_ can then be extended into _Better Docs_ with greater depth, quality and maturity.
+* Project outlines
+* Templates 
+* Best practices
 
-### Why?
+## Project outlines
 
-Providing writing templates:
+A project outline is a list of the guidebooks you need to create when you document your project -- it's really a list of templates that you can use. The list is different depending on what type of software you're documenting, because a RESTful API, for example, has different documentation requirements from desktop software or a browser-based app.
 
-*   Amplifies the writing effectiveness of developers, power users writers, and more.
-*   Improves doc quality and consistency across the board.
-*   Facilitates the democratisation of knowledge.
-*   Leads to self-empowerment and the betterment of humankind.
+As an example, suppose you have built a new RESTful API service. You could create reference documentation using Swagger and pat yourself on the back. But you could go further and complete the Minimum Viable Docset for a REST API: 
```

Comment:
would this information flow into one of @cwcromwell `about-api-overview.md` style artilces? There seems like there could be a bit of duplication here.

This is only a terse review from what I've seen while converting @cwcromwell templates so don't take my word for it.